### PR TITLE
Add M16 Cameras

### PR DESCRIPTION
**Pull request by Camasher**

### CODE REVIEW

File: assets/cameras.csv [hunk 1]
```diff
@@ -27,6 +27,8 @@ Mobotix AG,M15,10.18.168.61,16016,
 Mobotix AG,M15,10.7.46.153,16028,
 Mobotix AG,M15,10.20.245.115,16033,
 Mobotix AG,M15,10.20.210.120,,
+Mobotix AG,M16,10.26.62.41,12902,
+Mobotix AG,M16,TBA,12903,
```
Comment (ozym) [inline assets/cameras.csv:31]: might be best to add this when the serial is known, it will need another PR in any case.

Comment (Camasher) [reply assets/cameras.csv]: @ozym, serial number now added

**Comment by ozym** [comment]
Looks like the asset numbers have been used before?

**Comment by Camasher** [comment]
@ozym ah shouldn't have been, but maybe. I asked @Notashtonkutcher to grab me two numbers, might have been from a used pile

**Comment by ozym** [comment]
Sercel sensor number 642 etc.

**Comment by Notashtonkutcher** [comment]
LGTM

**Comment by Camasher** [comment]
Will need a Delta authorised person to merge @staylorofford

**Pull request merged**